src/notifications/templates/notifications/email/subscription_payment_failed.html

```
@@ -0,0 +1,44 @@
+{%load i18n %}<!DOCTYPE html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)